lib/timetable-visualisation/PlanViewer.tsx

```diff
@@ -46,33 +47,38 @@ export const PlanViewer = ({
     throw new Error("published event not found");
   }
 
-  const stagesInfo = useMemo<StagePreviewInfo[]>(
-    () => [
-      {
-        name: "Local Development Scheme Published",
-        startEvent: publishedEvent,
-      },
-      ...stages.map<StagePreviewInfo>((stage) => {
+  const stagesInfo = useMemo<StagePreviewInfo[]>(() => {
+    const foundStages = stages.reduce(
+      (allStages: StagePreviewInfo[], currentStage) => {
         const startEvent = timetableEvents.find(
-          (event) => event.developmentPlanEvent === stage.startEventKey
+          (event) => event.developmentPlanEvent === currentStage.startEventKey
         );
 
         const endEvent = timetableEvents.find(
-          (event) => event.developmentPlanEvent === stage.endEventKey
+          (event) => event.developmentPlanEvent === currentStage.endEventKey
         );
 
-        if (!startEvent) {
-          throw new Error("event not found");
+        if (startEvent && startEvent.eventDate) {
+          allStages.push({
+            name: currentStage.title,
+            startEvent,
+            endEvent,
+          });
         }
-        return {
-          name: stage.title,
-          startEvent,
-          endEvent,
-        };
-      }),
-    ],
-    [publishedEvent, timetableEvents]
-  );
+        return allStages;
+      },
+      []
+    );
+
+    if (publishedEvent && publishedEvent.eventDate) {
+      foundStages.push({
+        name: "Local Development Scheme Published",
+        startEvent: publishedEvent,
+      });
+    }
```

Comment:
Has this event deliberately been moved to the bottom of the visualisation?

Reply:
Also, to note, we're still throwing on line 47 if the LDS published event isn't present

Reply:
no actually I forgot that we want to display that one first. I will move it to the top 👍

Reply:
and I will remove the throwing there as well

Reply:
done! :)